hunt/app/views/animals/index.html.erb

```diff
@@ -1,35 +1,70 @@
- <div class="container">
-   <div class="row">
-     <div class="col">
-       <%= link_to 'New Animal', new_animal_path, class: 'btn btn-dark' %>
-       <table class="table table-hover mt-2 table-clickable">
-          <thead class="thead-dark">
-            <tr>
-              <th scope="col">Name</th>
-              <th scope="col">State</th>
-              <th scope="col">Category</th>
-            </tr>
-          </thead>
-          <tbody>
-
-
-            <% @animals.each do |animal| %>
-
-              <tr onclick="displayAnimal('<%=animal_path(animal)%>')">
-                <td><%=animal.name%></td>
-                <td><%=animal.state%></td>
-                <td><%=animal.category%></td>
-              </tr>
-              
-            <% end %>
-            
-          </tbody>
-        </table>
+<div class="container">
+  <div class="row">
+    <div class="col">
+
+      <div class="card">
+        <div class="card-body bg-light">
+          <%= simple_form_for :search, url: root_path, method: "get" do |f| %>
+          <div class="row">
+           
+           <div class="col-sm">
+               <%= f.input :name, required:false, input_html: {value: params.dig(:search, :name)}%>
+           </div>
+           
+           <div class="col-sm">
+             <%= f.input :state, collection:Animal::STATES, selected: params.dig(:search, :state), required:false, include_blank:true %>
+           </div>
+           
+           <div class="col-sm">
+             <%= f.input :category, collection: Animal::CATEGORIES,selected: params.dig(:search, :category), required:false, include_blank: true %>
+           </div>
+           
+         </div>
+         
+       <div class=row>
+         <div class="col-sm d-flex justify-content-start">
+           <%= link_to 'New Animal', new_animal_path, class: 'btn btn-dark' %>
+         </div>
+         
+         <div class="col-sm d-flex justify-content-end">
+          <%= link_to "Reset", animals_path, class: "btn btn-light" %>
+          <%=f.submit 'Search', class: 'btn btn-dark'%>
+        </div>
+       <% end %>
+        </div>
```

Comment:
In think this `</div>` should come before the `<% end %>` since it goes with the row inside the form. 
```suggestion
        </div>
       <% end %>
```